CMakeLists.txt

```diff
@@ -43,30 +56,62 @@ set(SERENITY_SOURCES
 )
 
 set(PIPELINE_TEST_SOURCES
-    pipeline/pipeline_test.cpp
+    tests/integration/pipeline_test.cpp
 )
 
-set(SERENITY_TEST_SOURCES
+set(SERENITY_UT_TESTS_SOURCES
```

Comment:
1) UT??
You mean, unit tests? So maybe "SERENITY_UNIT_TESTS_SOURCES"

2) Please order this set alphabetically